Tasks/JenkinsQueueJobV2/task.json

```diff
@@ -14,7 +14,7 @@
     "version": {
         "Major": 2,
         "Minor": 144,
```

Comment:
Should this be updated to 145?

Reply:
I don't think so, this is against release branch right?  But we probably should update the patch number?

Reply:
Yes, this is against the release branch. I'll post the PR against master directly. :)